cmd/vicadmin/logs.go

```diff
@@ -378,14 +378,15 @@ func findSeekPos(f *os.File) int64 {
 		// Go back nBytes from the last newline we've seen (or beginning-of-file),
 		// and read the next nBytes
 		readPos := lastnl - int64(len(buf))
-		if pos < 0 {
+		// Fix here.  Previously was "pos < 0".
```

Comment:
Yeah, we don't want to try reading from a negative offset.  That was generating an error. I'll nix that comment after the rest of the comments come through.